decidim-core/app/models/decidim/user.rb

```diff
@@ -11,7 +11,7 @@ class User < ApplicationRecord
 
     ROLES = %w(admin moderator official).freeze
 
-    validates :organization, presence: true
+    validates :organization, :name, presence: true
```

Comment:
Well, there's always the risk of someone impersonating someone else by putting its name (it's what shows up in comments, etc). But let's delegate this to the future and beyond.